### ISSUE TITLE
ep3 -- refactor folder location of `writing`

### ISSUE THREAD

**Issue by tfoerst3r** [swcarpentry/shell-novice#1292]
[Link to the stated episode.](https://swcarpentry.github.io/shell-novice/03-create/index.html)

During this episode, a small 'error' is placed in the description.

At the beginning of the lesson, you navigate to the folder `exercise-data/writing/` where you also create a folder `thesis`

Leading to:

~~~
$ ls -lF ~/desktop/shell-lesson-data/exercise-data/writing/
haiku.txt  LittleWomen.txt  thesis/
~~~

But in sections ***Moving files and directories*** and ***Removing files and directories*** it seems the folder `writing` moved one level up.

Leading to (I assume)

~~~
$ ls -lF ~/desktop/shell-lesson-data/writing/
haiku.txt  LittleWomen.txt  thesis/
~~~

This description of `$ cd ~/Desktop/shell-lesson-data/exercise-data/writing` switches back in section ***Operations with multiple files and directories***. 

It seems there only need to be made minor changes to the above mentioned two sections. It was a bit confusing during the preparation of a shell course.

Thx.
tfoerst3r

**Comment by gcapes** [comment]
Thanks for reporting this. I think I've fixed it in #1293.

**Comment by tfoerst3r** [comment]
Yes. You are right.
